src/imitation/util/networks.py

```diff
@@ -155,7 +155,7 @@ def __init__(
 
         self.decay = decay
 
-    def update_stats(self, batch: th.Tensor) -> None:
+    def update_stats_incremental(self, batch: th.Tensor) -> None:
```

Comment:
Do we expect this to be used for any purpose other than testing? If not, I'm inclined to pull it out of here. You could e.g. make a class in the test file that subclasses `EMANorm` and overrides `update_stats` in this way, or even just create two instances of `EMANorm` and use `mock.patch` to replace the `update_stats` method in one of them with this.

Reply:
Going ahead with the first approach.